Mosaicstg/climate-united
# Phase 2 - Team Member Pages

- Created team member frontend page
- Created url path for team member
- Updated Team Member bucket with read more link

Testing:

- Go to "Our People" page
- Navigate to a team member and click "read more"
- Check SEO titles are working for page without dedicated SEO
